_rules/image-non-empty-accessible-name-23a2a8.md

```diff
@@ -26,6 +26,8 @@ input_aspects:
   - CSS Styling
 acknowledgments:
   authors:
+    - Wilco Fiers
+  previous_authors:
```

Comment:
I thought this was necessary because of the applicability change, would like a yay or nay on that.

Reply:
Yay!

Reply:
No real opinion on that.
Maybe we should have some clear guidelines on the topic at some point 😄